std/js/Browser.hx

```diff
@@ -26,6 +26,12 @@ import js.html.Storage;
 import js.html.XMLHttpRequest;
 
 class Browser {
+	/** The global scope with additional fields available only in a worker context. */
+	public static var self(get, never):js.html.WorkerGlobalScope;
+
+	extern inline static function get_self()
```

Comment:
Add a type hint for return type. 
Otherwise it is typed as a monomorph (because of `untyped` return) and could possibly be bound to whatever first type it gets unified with.